README.md

```diff
@@ -1,70 +1,73 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Stocks Metrics
 
-## Available Scripts
+> A React-Redux stock metrics application.
+Get the best-performing stocks daily see stocks prices, their changes, and the most useful information of each stock.
 
-In the project directory, you can run:
 
-### `npm start`
+![screenshot](./img/mobileVersion.png)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Live Demo
```

Comment:
- [ ] Let's make sure you record video of your project as requested in project project guidelines.